packages/frontend/src/stdlib/analyses/decapodes.tsx

```diff
@@ -195,7 +195,10 @@ export function Decapodes(props: DiagramAnalysisProps<DecapodesContent>) {
                 </IconButton>
             </Match>
             <Match when={kernel.error || options.error}>
-                <IconButton onClick={restartKernel} tooltip="Restart the AlgebraicJulia service">
+                <IconButton
```

Comment:
Since the user is gonna have to know about kernels anyway, perhaps it's clear to replace "AlgebraicJulia Interop service" with simply "Julia kernel" here and below?